descriptors/ModuleDescriptor-template.json

```diff
@@ -1074,6 +1074,21 @@
           "permissionsRequired": ["inventory-storage.oai-pmh-view.enrichedinstances.collection.post"]
         }
       ]
+    },
+    {
+      "id": "inventory-hierarchy",
+      "version": "1.0",
```

Comment:
We usually use `0.1` version for new interfaces.

Reply:
Will fix it, thank you.